### PR TITLE
Ghana Tech Summit - Capstone 1

### PR DESCRIPTION
**Pull request by Kwentsir**
In this capstone project, the following features were implemented:

- Mobile: up to 768px wide.
- Desktop: 768px or wider.
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.
- The section "Featured speakers" was created dynamically in JavaScript.

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    />
+    <link rel="stylesheet" href="assets/css/popup.css" />
+    <title>Ghana Tech Summit</title>
+  </head>
+  <body>
+    <main>
+      <section class="homepage">
+        <header class="nav" id="header">
+          <div class="nav-container">
+            <a class="nav-logo" href="#">
```
Comment (Lordkaito) [inline popup.html]: - [ ] Please make sure that you can go back to your home page whenever you click on your logo, as right now it has no link to do so, so please work on this issue.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            </article>
+          </div>
+          <button type="button" id="join">
+            Register Ghana Tech Summit 2022 for free
+          </button>
+        </div>
+      </section>
+      <section class="homepage-speakers" id="speakers">
+        <div class="speakers-container">
+          <h2>Featured Speakers</h2>
```
Comment (Lordkaito) [inline index.html:176]: - [ ] In this section, if you check the original Figma template, you will realize that there is a little grid on each image of the featured speakers, so please, in order to stick to the given design as much as possible, implement this feature as well.

File: popup.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                  <h6>Logout</h6>
+                </a>
+              </div>
+            </div>
+            <nav>
+              <img
+                src="./assets/icons/menu.svg"
+                class="nav-menu"
+                alt="Menu Options"
+              />
```
Comment (Lordkaito) [inline popup.html:59]: - [ ] It looks like there is an issue with your JavaScript event for the hamburger menu on the about page. Right now, this event only works for the index page, so whenever I go to the about page and try to open it, it is not showing up. Take a look at your routing or any other possible issue so you can fix this.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Kwentsir, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (AmaduKamara) [reply index.html:176]: - [ ] Kindly work on this change as requested by the previous reviewer. From the design given, there is a grid floated design at the top left of each speakers' image. Kindly fix that.

**In the design below I marked the sections in green circles so you can spot it.
image

**Your Design**
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi, @Kwentsir,

AlonzoLeroneThisCloseGIF

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight
✅ No linter errors
✅ Good commit messages
✅ Good descriptive PR messages

## Required Changes

- [ ] Kindly look at the comment made in the previous review on the grid for the speakers' section. Please fix that and your project will be good for approval.

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

```

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Kwentsir,

ClappingLeonardoDicaprioGIF

Good job on making the changes I previously asked you to work on. 
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**